source/_components/switch.switchmate.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: page
+title: "Switchmate"
+description: "Instructions on how to have Switchmate switches."
+date: 2018-08-17 22:41
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: switchmate.png
```

Comment:
The name doesn't match the uploaded image.

Reply:
Opps